boards/arduino-mega2560/include/periph_conf.h

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2014 Freie Universität Berlin, Hinnerk van Bruinehsen
+ * Copyright (C) 2014-2016 Freie Universität Berlin
```

Comment:
Actually this code comes from Hauke, I didn't modify anything on this so I think is ok